lib/compat/wordpress-6.0/class-wp-rest-block-patterns-controller.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * REST API: WP_REST_Block_Patterns_Controller class
+ *
+ * @subpackage REST_API
+ * @package    WordPress
+ */
+
+/**
+ * Core class used to access block patterns via the REST API.
+ *
+ * @see   WP_REST_Controller
+ */
+class WP_REST_Block_Patterns_Controller extends WP_REST_Controller {
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->namespace = '__experimental';
+		$this->rest_base = 'block-patterns/patterns';
+	}
+
+	/**
+	 * Registers the routes for the objects of the controller.
+	 *
+	 * @see   register_rest_route()
+	 */
+	public function register_routes() {
+		register_rest_route(
```

Comment:
Is there anyway to get a single block pattern? Like `__experimental/block-patterns/patterns/<id>`